.travis.yml

```diff
@@ -45,12 +45,6 @@ after_failure:
         - tail -100 _build/mim2/rel/mongooseim/log/ejabberd.log
```

Comment:
I guess this file should be completely removed.

Reply:
what do you mean ?

Reply:
I disagree, it is sometimes helpful when there is no proper logs upload in place (e.g. on forks).

Reply:
I meant `.travis.yml` :)

Reply:
Argh, right, file, not line.
Still, we want it keep it for the time being until we decide that CircleCI works 100% fine and we want to abandon Travis. :)

Reply:
So we'll be running both at the same time? Because if not, we can always revert the file, there's no need to keep it unless we really want to run builds on both at the same time.